Tests/PubNubTests/Mocking/MockURLSession.swift

```diff
@@ -133,6 +217,34 @@ class MockURLSession: URLSessionReplaceable {
     return task
   }
 
+  func uploadTask(withStreamedRequest request: URLRequest) -> URLSessionUploadTask {
+    guard let task = self.state.lockedRead({ $0.tasks.first { $0.originalRequest == request } }) as? URLSessionUploadTask else {
```

Comment:
Codacy found an issue: [Line should be 120 characters or less: currently 128 characters](https://app.codacy.com/manual/PubNub/swift/pullRequest?prid=6302911)